PMDMMO-common/src/com/darkxell/common/ai/visibility/Visibility.java

```diff
@@ -98,6 +100,16 @@ public void onPokemonMoved() {
         }
     }
 
+    /** @return The list of enemy Pokemon the input Pokemon can see, sorted by distance to the Pokemon. */
```

Comment:
Is there anything in this method that ensures that it is sorted by distance to Pokemon? If not, then I don't think you should make that claim here.

Reply:
My bad, I thought it was. I probably forgot one thing or two when migrating from AIUtils to Visibility